### ISSUE TITLE
Infinite loop on 3x rgb definition with linear-gradient

### ISSUE THREAD

**Issue by carlosjeurissen** [csstools/postcss-double-position-gradients#6]
The following CSS code:
```
background-image: linear-gradient(rgb(66 133 244) 19px, rgb(234 67 53) 20px, rgb(234 67 53) 30px);
```

Causes `postcss-double-position-gradients` to hang.

Might be related to: https://github.com/csstools/postcss-double-position-gradients/pull/5

**Comment by ehoogeveen-medweb** [comment]
This seems to have regressed in the just released version `3.0.2`. Explicitly installing `postcss-double-position-gradients@3.0.1` (which is normally an indirect dependency for us) fixes the hang.

**Comment by Antonio-Laguna** [comment]
@carlosjeurissen and @ehoogeveen-medweb thanks a lot for taking the time to report this issue. We're removing a library in all of our plugins (see csstools/postcss-preset-env#228) and it could create issues. There's now version 3.0.3 just published that deals with this and a new test case has been created to account for this in the future.

Thanks!
